lms/djangoapps/ccx/tests/test_utils.py

```diff
@@ -108,3 +114,69 @@ def test_add_master_course_staff_to_ccx(self):
             list_instructor_ccx_course = list_with_level(course_ccx, 'instructor')
             self.assertEqual(len(list_instructor_ccx_course), len(list_instructor_master_course))
             self.assertEqual(list_instructor_ccx_course[0].email, list_instructor_master_course[0].email)
+
+    def test_add_master_course_staff_to_ccx_display_name(self):
+        """
+        Test add staff of master course to ccx course.
+        Specific test to check that a passed display name is in the
+        subject of the email sent to the enrolled users.
+        """
+        outbox = self.get_outbox()
+        # create a unique display name
+        display_name = 'custom_display_{}'.format(uuid.uuid4())
+        list_staff_master_course = list_with_level(self.course, 'staff')
+        list_instructor_master_course = list_with_level(self.course, 'instructor')
+        self.assertEqual(len(outbox), 0)
+        # give access to the course staff/instructor
+        add_master_course_staff_to_ccx(self.course, self.ccx_locator, display_name)
+        self.assertEqual(len(outbox), len(list_staff_master_course) + len(list_instructor_master_course))
+        for email in outbox:
+            self.assertIn(display_name, email.subject)
+
+    def test_add_master_course_staff_to_ccx_idempotent(self):
+        """
+        Test add staff of master course to ccx course multiple time will
+        not result in multiple enrollments.
+        """
+        outbox = self.get_outbox()
+        list_staff_master_course = list_with_level(self.course, 'staff')
+        list_instructor_master_course = list_with_level(self.course, 'instructor')
+        self.assertEqual(len(outbox), 0)
+
+        # run the assignment the first time
+        add_master_course_staff_to_ccx(self.course, self.ccx_locator, self.ccx.display_name)
+        self.assertEqual(len(outbox), len(list_staff_master_course) + len(list_instructor_master_course))
+        with ccx_course(self.ccx_locator) as course_ccx:
+            list_staff_ccx_course = list_with_level(course_ccx, 'staff')
+            list_instructor_ccx_course = list_with_level(course_ccx, 'instructor')
+        self.assertEqual(len(list_staff_master_course), len(list_staff_ccx_course))
```

Comment:
Is it possible to compare the lists directly instead of the count, or is that cumbersome?